dbt_subprojects/hourly_spellbook/models/_sector/gas/fees/gas_fees.sql

```diff
@@ -83,6 +85,7 @@
     , "zkevm"
     , "zksync"
     , "zora"
+    , "plume"
```

Comment:
```suggestion
```

Reply:
looks like you accidentally put plume in here twice, this will lead to union'ing the data twice and have duplicate rows